engine/src/main/java/io/zeebe/engine/processing/common/CatchEventBehavior.java

```diff
@@ -189,6 +191,7 @@ private void subscribeToMessageEvent(
     subscription.setCorrelationKey(correlationKey);
     subscription.setTargetElementId(handler.getId());
     subscription.setCloseOnCorrelate(closeOnCorrelate);
+    // todo(zell): phil will migrate this via writing the open subscription
     workflowInstanceSubscriptionState.put(subscription);
```

Comment:
Can we already merge this PR without these changes?

Reply:
I would say yes. It is just for us to made the first step and not do it always again for the other processors